ansible/openshift-playbook.yml

```diff
@@ -115,7 +115,7 @@
         __test_harness_bc: linux-system-roles-centos7-staging
         __test_harness_secret_name: secrets-staging
         __test_harness_ghtoken_file: github-token-staging
-        __test_harness_deploy_replicas: 1
+        __test_harness_deploy_replicas: "{{ test_harness_deploy_replicas_staging | d(1) }}"
```

Comment:
What about 4 variables for replicas?? test_harness_deploy_replicas_staging, test_harness_deploy_replicas,  **test_harness_deploy_replicas_centos_staging, test_harness_deploy_centos_replicas** to make it more configurable? 

@richm

Reply:
I am going to merge the PR as it is. The change you suggest can be done in a future PR.